workspace/viewer/PdfJsViewer.ts

```diff
@@ -451,9 +451,9 @@ export class PdfJsViewer {
           textLayerFactory: this.textLayerFactory
         })
         this.loadedPages.push(pageView);
+        pageView.update(scale);
+        pageView.setPdfPage(page);
```

Comment:
What's the reason of this change? You no longer call `setPdfPage` in case of resize -- is it just not necessary?